packaging/rpm/microshift.spec

```diff
@@ -166,6 +166,9 @@ install -p -m644  packaging/network-manager-conf/microshift-nm.conf %{buildroot}
 install -d -m755 %{buildroot}/%{_unitdir}
 install -p -m644 packaging/systemd/microshift.service %{buildroot}%{_unitdir}/microshift.service
 
+install -d -m755 %{buildroot}/%{_sysconfdir}/microshift
+install -p -m644 packaging/microshift/microshift.yaml %{buildroot}%{_sysconfdir}/microshift/microshift.yaml
```

Comment:
I wasn't sure whether to install the file to the real location or to an "example.yaml" location. I decided to go for the real location, but I'm open to changing that.